regression/contracts/invar_assigns_alias_analysis/test.desc

```diff
@@ -0,0 +1,14 @@
+KNOWNBUG
+main.c
+--apply-loop-contracts
+^EXIT=0$
+^SIGNAL=0$
+^\[main.1\] .* Check loop invariant before entry: SUCCESS$
+^\[main.2\] .* Check that loop invariant is preserved: SUCCESS$
+^\[main.assertion.1\] .* assertion b->data[5] == 0: FAILURE$
+^VERIFICATION FAILED$
+--
+--
+This test shows that the current implementation only supports
+parsing and still uses alias analysis. Fixing this is a part 
```

Comment:
What do you mean by "current implementation only supports parsing and still uses alias analysis"? Could you elaborate?

Reply:
This PR only adds parsing to it, and assertion statements are yet not generated. Therefore, this proof does not go through.

Reply:
What Aalok is saying is that, he added this test just to check the parsing support (and address Martin's comment regarding usage of assigns clause). But it's not considered during instrumentation, so this test which he obtained from my bug report, still fails as before.